js/__loader.js

```diff
@@ -283,6 +288,22 @@
 
   global.process = loader.require('process');
   global.Buffer = loader.require('buffer').Buffer;
+  const stream = loader.require('stream');
+  class StdoutStream extends stream.Writable {
+    _write(chunk, encoding, callback) {
+      runtime.stdio.defaultStdio.write(String(chunk));
+      callback();
+    }
+  }
+  class StderrStream extends stream.Writable {
+    _write(chunk, encoding, callback) {
+      runtime.stdio.defaultStdio.writeError(String(chunk));
+      callback();
+    }
+  }
```

Comment:
@facekapow why does this file contains es5 and es6 style both? looks odd. Is this okay?